arch/arm/src/phy62xx/uart.c

```diff
@@ -921,11 +941,21 @@ static void pplus_uart_rxint(struct uart_dev_s *dev, bool enable)
 
 static bool pplus_uart_rxavailable(struct uart_dev_s *dev)
 {
+  static int len_fifo;
   uart_Ctx_t *priv = (uart_Ctx_t *)dev->priv;
-
-  /* Return true if the receive buffer/fifo is not "empty." */
-
-  return priv->rx_available;
+  /* Detect the length of received data from fifo*/
+  if(len_fifo == 0)
+  {
+  	  hal_UART0_IRQHandler();
+  	  len_fifo = fifo_len_store;
+  }
+  if(len_fifo <= 0)
+	  {
+	  len_fifo = 0;
+  	  return len_fifo;
+	  }
+  else
+	  return len_fifo--;
```

Comment:
fix the nxstyle warning

Reply:
i have already runed checkpatch.sh , this part is correct.
not find any nx style warning.